vdb-bench-assembly/plugins/vdb-bench-dataservice/export/dsExportGitWizard.js

```diff
@@ -90,7 +103,25 @@
                 return false;
             }
 
-            return true;
+            if ( $rootScope.requireAuthorName && _.isEmpty(vm.repo.parameters['author-name-property'])) {
+                return false;
```

Comment:
I see the sense of doing this and understand how it solve the difficulty. Yet I can still hear my former programming tutor at uni declaring me as Satan for suggesting the use of global variables. Maybe we can stick a TODO comment around these to maybe revisit in the future to see if there is an alternative way?